packages/routecraft/README.md

```
@@ -32,6 +32,227 @@ export default craft()
+    return {
+      model,
+      totalTokens: inputTokens + outputTokens,
+      cost: calculateCost(model, inputTokens, outputTokens)
+    };
+  })
+  .to(log());
+```
+
+### Complete Event Hierarchy
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)